visualizer/index.html

```diff
@@ -203,7 +312,7 @@
     #bottomSection {
       display: flex;
       flex: 1;
-      flex-direction: column;
```

Comment:
Trying to list zoomOutButton, parse result, and action buttons side by side, so there won't be too much vertical spaces taken by bottoms.